mscore/shortcut.cpp

```diff
@@ -237,8 +237,8 @@ Shortcut Shortcut::_sc[] = {
          STATE_NORMAL | STATE_NOTE_ENTRY,
          "note-input",
          QT_TRANSLATE_NOOP("action","Note Input"),
+         QT_TRANSLATE_NOOP("action","Note input: Toggle Entry/Normal mode"),
          QT_TRANSLATE_NOOP("action","Note input"),
```

Comment:
this line doesn't add any value. Better remove it

Reply:
Do you mean exactly this "Note input" entry or all these third entries in shortcuts list?

Reply:
Just this (I think)

Reply:
It is a tooltip IIRC, which here, as just repeating information, isn't needed at all